src/pybind/mgr/cephadm/module.py

```diff
@@ -3114,7 +3115,7 @@ def generate_certificates(self, module_name: str) -> Optional[Dict[str, str]]:
         if module_name not in supported_moduels:
             raise OrchestratorError(f'Unsupported modlue {module_name}. Supported moduels are: {supported_moduels}')
 
-        host_fqdns = [socket.getfqdn(self.get_hostname())]
+        host_fqdns = [self.get_hostname()]
```

Comment:
plz, don't change this in this PR... it will be fixed in other PR

Reply:
Removed